frontend/src/pages/offers/LineItemForm.tsx

```diff
@@ -147,6 +150,14 @@ const LineItemForm: FunctionComponent<Props> = ({
             options={LINE_ITEM_CONTAINER_OPTIONS}
             register={register}
             required
+            registerOptions={{
+              validate: {
+                notUnset: (value: LineItemContainerType) =>
+                  value !== LineItemContainerType.Unset ||
+                  'Please select a container type',
+              },
```

Comment:
`UNSET` is a choice in the dropdown but we don't want to allow users to select it